src/test/java/org/opentripplanner/middleware/utils/ItineraryUtilsTest.java

```diff
@@ -129,13 +116,13 @@ void canCheckAllItinerariesExist(boolean insertInvalidDay, String message) throw
         mockOtpResponses.set(FRIDAY_INDEX, OtpTestUtils.OTP_DISPATCHER_PLAN_ERROR_RESPONSE.getResponse());
         mockOtpResponses.set(WEDNESDAY_INDEX, OtpTestUtils.OTP_DISPATCHER_PLAN_ERROR_RESPONSE.getResponse());
 
-        OtpTestUtils.setupOtpMocks(mockOtpResponses);
+        MockOtpResponseProvider mockResponses = new MockOtpResponseProvider(mockOtpResponses);
```

Comment:
Perhaps out of scope, but would other classes benefit from this change and aid future testing? Five test classes e.g. `TrackedTripControllerTest` still use `OtpTestUtils.mockOtpServer()`.

Reply:
I agree. Hopefully we can go one at a time.